src/ts/components/seekbar.ts

```diff
@@ -65,6 +65,10 @@ export class SeekBar extends Component<SeekBarConfig> {
    * The CSS class that is added to the DOM element while the seek bar is in 'seeking' state.
    */
   private static readonly CLASS_SEEKING = 'seeking';
+  /**
+   * Offset for the translateX/translateY value applied when scrubbing in %
+   */
+  private static TRANSLATE_OFFSET = -50;
```

Comment:
This value is only used once and the description is not really explaining anything as there's no added value in creating this field... better inline it directly.